model/github_runner.rb

```diff
@@ -9,6 +9,10 @@ class GithubRunner < Sequel::Model
 
   include ResourceMethods
 
+  def self.redacted_columns
+    super + [:workflow_job]
+  end
```

Comment:
Should we commit this separately and, like, now? this redacted columns list has other uses, e.g. in inspect.

Reply:
https://github.com/ubicloud/ubicloud/pull/1030